doc/release-notes/release-7.sgml.in

```diff
@@ -125,6 +125,25 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing directives<label id="modifieddirectives">
 <p>
 <descrip>
+	<tag>acl</tag>
+	<p>Changed <em>src</em> to detect and handle overlapping IP and
```

Comment:
IMHO, #1632 _fixes_ how these seven ACLs are parsed. I recommend using the word "fixed" rather than the word "changed". I do not insist on this.